main.c

```diff
@@ -1255,10 +1259,11 @@ uploadsnap(const char * fname, const char * imgfmt, const char * region,
 		goto err1;
 	}
 	len = sb.st_size;
+	*imgsz = len;
 
 	/* Map the file. */
 	if ((p = mmap(NULL, len, PROT_READ, MAP_PRIVATE | MAP_NOCORE,
-	    fileno(f), 0)) == MAP_FAILED) {
+		fileno(f), 0)) == MAP_FAILED) {
```

Comment:
Why did you change the indentation here?

Reply:
I have no idea how this happened